src/python/pants/core/goals/test.py

```diff
@@ -91,6 +80,32 @@ def skipped(self) -> bool:
             and not self.xml_results
         )
 
+    def __lt__(self, other: Union[Any, "EnrichedTestResult"]) -> bool:
+        """We sort first by status (skipped vs failed vs succeeded), then alphanumerically within
```

Comment:
Nit: Docstring doesn't conform to https://www.python.org/dev/peps/pep-0257/ (first sentence must fit on a single line).